### PR TITLE
Updating README.msi.md to account for GA behavior, and to specify min-access requirements

### LINKED ISSUE #534: README.msi.md should call out min-access permissions instead of full Contributor
**Describe the request**
The [README.msi.md](https://github.com/Azure/aad-pod-identity/blob/master/docs/readmes/README.msi.md) document explains that you need to assign full Contributor access to the user-assigned identity for the MC_ resource group.  This allows much more capability than is required, and I suspect it was documented this way more as a convenience.

**Explain why AAD Pod Identity needs it**
To promote good security hygiene for aad-pod-identity users! :)

**Describe the solution you'd like**
Update the document to explain min-access requirements are: "Virtual Machine Contributor" and "Managed Identity Operator" on the MC_ resource group.  (Determined by myself with some trial and error)

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by DrEsteban**
**Reason for Change**:

Update MSI documentation to account for updated AKS behavior now that the functionality is GA, and also specifying min-access requirements instead of full Contributor access.

**Issue Fixed**:

Fixes #534 

**Notes for Reviewers**:
I also updated the command for showing the user-assigned MSI because now AKS surfaces that information natively.  Instead of getting it from the VMAS/VMSS.

### CODE REVIEW

File: docs/readmes/README.msi.md [hunk 1]
```diff
@@ -17,25 +17,19 @@ The rest of the README describes the prerequisite role assignments to be perform
...
-for VMAS:
-
-```bash
-az vm identity show -g <resource group> -n <vm name> -o yaml
-```
-for VMSS:
+After the cluster is created, run this command to retrieve the cluster's user assigned principal id:
 ```bash
-az vmss identity show -g <resource group>  -n <vmss scalset name> -o yaml
+az aks show -g <resource group> -n <aks cluster name> --query identityProfile.kubeletidentity.clientId -o tsv
```
Comment (aramase) [inline docs/readmes/README.msi.md]: This command works for AKS clusters, but we also need to keep the `az vmss identity show` command for aks-engine created clusters. So can we keep the old command and also this command for AKS clusters?

File: docs/readmes/README.msi.md [hunk 2]
```diff
@@ -17,25 +17,19 @@ The rest of the README describes the prerequisite role assignments to be perform
...
 
-The type in the output of the above command will identify the system assigned or user assigned MSI. Please record the corresponding
-principal id.
+The type in the output of the above command will identify the user assigned MSI. Please record the principal id.
 
-For creating a role assignment to authorize assignment/removal of user assigned identities on VMS/VMSS, run the following command:
+For creating a role assignment to authorize assignment/removal of user assigned identities on VMS/VMSS, run the following commands:
 ```bash
-az role assignment create --role "Contributor" --assignee <principal id from az vm/vmss identity command>  --scope /subscriptions/<sub id>/resourcegroups/<resource group name>
+az role assignment create --role "Virtual Machine Contributor" --assignee <principal id from az aks show command>  --scope /subscriptions/<sub id>/resourcegroups/<resource group name>
```
Comment (aramase) [inline docs/readmes/README.msi.md]: Thank you for this!

**Comment by DrEsteban** [comment]
Roger that! Will update.

File: docs/readmes/README.msi.md (hunk 1 above)
Comment (DrEsteban) [reply docs/readmes/README.msi.md]: Applied these changes, let me know what you think!

**Comment by DrEsteban** [comment]
@aramase Updated PR

**Comment by aramase** [comment]
Thank you @DrEsteban! 🎉

**Pull request merged**